parking_lot_src/ParkingLot/ParkingLot.java

```diff
@@ -0,0 +1,104 @@
+package ParkingLot;
+
+import java.util.*;
+
+public class ParkingLot {
+    int floorCount;
+    String id;
+    int slots_per_floor;
+    List<Floors> floorsList = new ArrayList<Floors>();
+    HashMap<String,Vehicles> vehicleTicketMap = new HashMap<String,Vehicles>();
+
+    public ParkingLot(int floorCount, String id, int slots_per_floor) {
+        this.floorCount = floorCount;
+        this.id = id;
+        this.slots_per_floor = slots_per_floor;
+
+        for(int i=0;i<floorCount;i++) {
```

Comment:
Move this out from constructor

Reply:
This file should move under service package, Anything where you are writing business logic, that should move under service package